DataConnectors/Imperva WAF Gateway/Connector_Imperva_WAF_Gateway.json

```diff
@@ -0,0 +1,119 @@
+        "innerSteps": [
+            {
+                "title": "1.1 Select or create a Linux machine",
+                "description": "Select or create a Linux machine that Azure Sentinel will use as the proxy between your security solution and Azure Sentinel this machine can be on your on-prem environment, Azure or other clouds."
+            },
+            {
+                "title": "1.2 Install the CEF collector on the Linux machine",
+                "description": "Install the Microsoft Monitoring Agent on your Linux machine and configure the machine to listen on the necessary port and forward messages to your Azure Sentinel workspace. The CEF collector collects CEF messages on port 514 TCP.\n\n> 1. Make sure that you have Python on your machine using the following command: python -version.\n\n> 2. You must have elevated permissions (sudo) on your machine.",
+                "instructions": [
+                    {
+                        "parameters": {
+                            "fillWith": [
+                                "WorkspaceId",
+                                "PrimaryKey"
+                            ],
+                            "label": "Run the following command to install and apply the CEF collector:",
+                            "value": "sudo wget -O cef_installer.py https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"
+                        },
+                        "type": "CopyableLabel"
+                    }
+                ]
+            }
+        ]
+    },
+    {
+        "title": "2. SecureSphere MX Configuration",
+        "description": "This connector requires an Action Interface and Action Set to be created on the Imperva SecureSphere MX.  [Follow the steps](https://community.imperva.com/blogs/craig-burlingame1/2020/11/13/steps-for-enabling-imperva-waf-gateway-alert) to create the requirements.",
+        "innerSteps": [
+            {
+                "title": "2.1 Create the Action Interface",
+                "description": "Create a new Action Interface that contains the required parameters to send WAF alerts to Azure Sentinel."
+            },
+            {
+                "title": "2.2 Create the Action Set ",
+                "description": "Create a new Action Set that uses the Action Interface configured."
+            }
+        ]
+    },
+    {
+        "title": "3. Apply the Action Set",
+        "description": "Apply the Action Set to any Security Policies you wish to have alerts for sent to Azure Sentinel."
+    },
+    {
+        "title": "4. Validate connection",
+        "description": "Follow the instructions to validate your connectivity:\n\nOpen Log Analytics to check if the logs are received using the CommonSecurityLog schema.\n\n"
```

Comment:
Instructions for connection validation are missing. Please refer template https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json.